### PR TITLE
Chore: Exporting definitions & updating the action

### PR DESCRIPTION
**Pull request by aarongustafson**
Appeasing respec

### CODE REVIEW

**Comment by aarongustafson** [comment]
@marcoscaceres I think I got the updates to the action correct, but would appreciate another set of eyes.

File: index.html [hunk 1]
```diff
@@ -105,7 +105,7 @@ <h3>
           `categories` member
         </h3>
         <p>
-          The <code><dfn>categories</dfn></code> member is an <a>array</a> of
+          The <code><dfn class="export">categories</dfn></code> member is an <a>array</a> of
```
Comment (marcoscaceres) [inline index.html:108]:
```suggestion
          The <code><dfn data-dfn-for="manifest" class="export">categories</dfn></code> member is an <a>array</a> of
```

File: index.html [hunk 2]
```diff
@@ -193,7 +193,7 @@ <h3>
           `description` member
         </h3>
         <p>
-          The <code><dfn>description</dfn></code> member is a <a>string</a>
+          The <code><dfn class="export">description</dfn></code> member is a <a>string</a>
```
Comment (marcoscaceres) [inline index.html:196]:
```suggestion
          The <code><dfn data-dfn-for="manifest" class="export">description</dfn></code> member is a <a>string</a>
```

File: index.html [hunk 3]
```diff
@@ -205,7 +205,7 @@ <h3>
           `iarc_rating_id` member
         </h3>
         <p>
-          The <code><dfn>iarc_rating_id</dfn></code> member is a <a>string</a>
+          The <code><dfn class="export">iarc_rating_id</dfn></code> member is a <a>string</a>
```
Comment (marcoscaceres) [inline index.html:208]:
```suggestion
          The <code><dfn data-dfn-for="manifest" class="export">iarc_rating_id</dfn></code> member is a <a>string</a>
```

File: index.html [hunk 4]
```diff
@@ -234,7 +234,7 @@ <h3>
           `screenshots` member
         </h3>
         <p>
-          The <code><dfn>screenshots</dfn></code> member is an array of
+          The <code><dfn class="export">screenshots</dfn></code> member is an array of
```
Comment (marcoscaceres) [inline index.html:237]:
```suggestion
          The <code><dfn data-dfn-for="manifest" class="export">screenshots</dfn></code> member is an array of
```

File: index.html [hunk 5]
```diff
@@ -253,7 +253,7 @@ <h3>
           `label` member
         </h3>
         <p>
-          The <code><dfn>label</dfn></code> member is a <a>string</a> that
+          The <code><dfn class="export">label</dfn></code> member is a <a>string</a> that
```
Comment (marcoscaceres) [inline index.html:256]:
```suggestion
          The <code><dfn data-dfn-for="manifest" class="export">label</dfn></code> member is a <a>string</a> that
```

File: index.html [hunk 6]
```diff
@@ -276,7 +276,7 @@ <h3>
           `platform` member
         </h3>
         <p>
-          The <code><dfn>platform</dfn></code> member is a <a>string</a> that
+          The <code><dfn class="export">platform</dfn></code> member is a <a>string</a> that
```
Comment (marcoscaceres) [inline index.html:279]:
```suggestion
          The <code><dfn data-dfn-for="manifest" class="export">platform</dfn></code> member is a <a>string</a> that
```

**Review by marcoscaceres — APPROVED** [review]
Couple of suggestions... just binding each definition to "manifest".

**Pull request merged**